spec/mysql/default_lambda/default_lambda_spec.rb

```diff
@@ -49,17 +49,17 @@
     before do
       subject.diff(erbh(<<-ERB)).migrate
         create_table "foos", force: :cascade do |t|
-          t.datetime "bar", <%= i cond(">= 7.0", { precision: nil }) %>, default: -> { "CURRENT_TIMESTAMP" }, null: false
+          t.datetime "bar", default: -> { "CURRENT_TIMESTAMP<%= i cond(">= 7.0", "(6)") %>" }, null: false
         end
       ERB
     end
 
     it do
-      delta = subject.diff(<<-RUBY)
+      delta = subject.diff(erbh(<<-ERB))
         create_table "foos", force: :cascade do |t|
-          t.datetime "bar", default: -> { '"1970-01-01 00:00:00"' }, null: false
+          t.datetime "bar", <%= i cond(">= 7.0", { precision: 6 }) %>, default: -> { '"1970-01-01 00:00:00"' }, null: false
```

Comment:
The test failed when I omitted this precision option.
I feel this behavior is odd 🤔